core/src/main/resources/hudson/model/Messages.properties

```
@@ -163,6 +163,9 @@ Hudson.AdministerPermission.Description=\
   This permission grants the ability to make system-wide configuration changes, \
   as well as perform highly sensitive operations that amounts to full local system access \
   (within the scope granted by the underlying OS.)
+
+Hudson.ConfigureJenkins.Description=Required to make non-sensitive configuration changes 
```

the Permission is `Jenkins.ConfigureJenkins` and I would expect this to be in a file in jenkins/model (but yes you have the fun of multiple imports of classes called `Messages`
the Message itself needs much more details and should fit in with the existing style of the other desctiptions.

eg. something like
```suggestion
Hudson.ConfigureJenkins.Description=\
 This permission grants the ability to make non Permission escalating system configuration changes.
 System configuration which would allow the execution of arbitrary commands or code on the master requires Administer
```

---

Agreed, this is a `WIP` and meant to be a restaging of some earlier work.  Expect more commits to follow.  @oleg-nenashev  - as the overall work will probably span multiple dependant PR's, what is the best process to follow?